.gitignore

```diff
@@ -99,10 +99,11 @@ hs_err_pid*
 # Ignore all .txt files in the directory doc/
 # doc/**/*.txt
 
-# Ignore .doc .pfd .docx
+# Ignore .doc .pfd .docx .json
```

Comment:
not ideal 
it's not great to have jsons in root dir, but potentially you can use those resources